go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/celestiaorg/knuu v0.10.0
 	github.com/celestiaorg/nmt v0.20.0
 	github.com/celestiaorg/rsmt2d v0.12.0
-	github.com/cosmos/cosmos-proto v1.0.0-alpha8
+	github.com/cosmos/cosmos-proto v1.0.0-beta.4
```

Comment:
There are no release notes but https://github.com/cosmos/cosmos-proto/tags doesn't describe anything breaking and protoc CI checks still pass so LGTM

Reply:
I share your viewpoint too.